### PR TITLE
Collection of small code cleanups

### PR DESCRIPTION
**Pull request by cfergeau**
Miscellaneous patches I've gathered over time.. Should not change much from a user perspective.

### CODE REVIEW

**Comment by praveenkumar** [comment]
CI is failing with
```
22:26:07     When checking that CRC is running                                                              # crcsuite.go:221 -> github.com/code-ready/crc/test/e2e/crcsuite.CheckCRCStatus
22:26:07     output did not match. Expected: '^OpenShift: .*Running \(v\d+\.\d+\.\d+.*\).*', Actual: 'CRC VM:          Running
22:26:07 OpenShift:       Running (v4.7.0)
22:26:07 Disk Usage:      11.3GB of 32.74GB (Inside the CRC VM)
22:26:07 Cache Usage:     10.78GB
22:26:07 Cache Directory: /home/crc_ci/.crc/cache'
```

File: test/e2e/crcsuite/crcsuite.go [hunk 1]
```diff
@@ -219,7 +219,7 @@ func CheckOutputMatchWithRetry(retryCount int, retryTime string, command string,
 // CheckCRCStatus checks that output of status command
 // matches given regex
 func CheckCRCStatus(state string) error {
-	expression := `.*Running \(v\d+\.\d+\.\d+.*\).*`
+	expression := `^OpenShift: .*Running \(v\d+\.\d+\.\d+.*\).*`
 	if state == "stopped" {
 		expression = ".*Stopped.*"
```
Comment (guillaumerose) [inline test/e2e/crcsuite/crcsuite.go:224]: Also add `Openshift: ` prefix here?

**Comment by jsliacan** [comment]
> CI is failing with
> 
> ```
> 22:26:07     When checking that CRC is running                                                              # crcsuite.go:221 -> github.com/code-ready/crc/test/e2e/crcsuite.CheckCRCStatus
> 22:26:07     output did not match. Expected: '^OpenShift: .*Running \(v\d+\.\d+\.\d+.*\).*', Actual: 'CRC VM:          Running
> 22:26:07 OpenShift:       Running (v4.7.0)
> 22:26:07 Disk Usage:      11.3GB of 32.74GB (Inside the CRC VM)
> 22:26:07 Cache Usage:     10.78GB
> 22:26:07 Cache Directory: /home/crc_ci/.crc/cache'
> ```

I think that's because 'OpenShift' isn't at the beginning of the string (it's a multi-line string). I wasn't paying much attention to this, but I think `^OpenShift` was only added recently. The `(v\d+\.\d+\.\d+.*\).*` already uniquely identified the line with OpenShift status, so `^OpenShift` shouldn't be needed?

**Comment by cfergeau** [comment]
> I think that's because 'OpenShift' isn't at the beginning of the string (it's a multi-line string). The `(v\d+\.\d+\.\d+.*\).*` already uniquely identified the line with OpenShift status, so `^OpenShift` shouldn't be needed (or we don't need to check that it's at the beginning of string)?

Ah good point, must be because it's multi-line! The test is working without the 'OpenShift' line, however, I feel it makes things clearer on failures.
```
Expected: '.*Running \(v\d+\.\d+\.\d+.*\).*', Actual: 'CRC VM:          Running
22:26:07 OpenShift:       Unknown
```
When seeing such a message, it's not clear if it's the `CRC VM:` line which is unexpected, or the `OpenShift` line. Adding `OpenShift` to the regexp removes this ambiguity.

File: test/e2e/crcsuite/crcsuite.go (hunk 1 above)
Comment (cfergeau) [reply test/e2e/crcsuite/crcsuite.go]: Indeed, fixed in the latest iteration.

**Pull request merged**